tests/container-scripts/run-testnet.sh

```diff
@@ -5,10 +5,16 @@ BIN=gravity
 
 NODES=$1
 set +u
-TEST_TYPE=$2
-ALCHEMY_ID=$3
+BINARY=$2
```

Comment:
test type should remain argument 2, right now this is breaking the tests in CI becuase TEST_TYPE is being interpreted as an old binary.